internal/pkg/skuba/upgrade/addon/version.go

```diff
@@ -61,35 +60,22 @@ func PrintAddonUpdates(updatedAddons AddonVersionInfoUpdate) {
 			continue
 		}
 
-		hasVersionUpdate := hasAddonVersionUpdateWithAddon(updatedAddons, addonName)
-		hasManifestUpdate := hasAddonManifestUpdateWithAddon(updatedAddons, addonName)
-		if hasVersionUpdate && !hasManifestUpdate {
+		// At this point we know that this addon has a greater manifest version, if the addon version
+		// is different than the one stored, we will show that to the user. If the versions are equals
+		// (string comparison), we will show the manifest version bump as additional information.
+		if updatedAddons.Current[addonName].Version != versions.Version {
```

Comment:
I think it should check manifest version first. Otherwise, it encounter user1 and user2 with different skuba version, always print out an addon need to upgrade

Reply:
Plugins that are inside the `updatedAddons.Updated` structure are the ones that already had a newer `ManifestVersion`. I could make the explicit check here as well, but it would be redundant.

Reply:
okay. I see it